DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -86,8 +86,8 @@ - (BOOL)isDead
 
 - (void)dealloc
 {
-	[self untrack];
```

Comment:
If I understand properly, the only time we should be `dealloc`ing is when the Trampoline is removed from `DTXTimerSyncResource _timers`?

For us to have done that, we must have called `untrack` already.

Reply:
That's what I understand as well. I will revisit this and make sure that's indeed the case but generally I think you're right.